packages/components/src/components/select-native/select-native.jsx

```diff
@@ -15,6 +15,16 @@ const getDisplayText = (list_items, value) => {
     return '';
 };
 
+const getOptions = (list_items, hide_disabled, use_text) => {
+    const options = hide_disabled ? list_items.filter(opt => !opt.disabled) : list_items;
+
+    return options.map((option, idx) => (
+        <option key={idx} value={use_text ? option.text : option.value}>
```

Comment:
Try to not use indexes as component keys if you don't have to (see line 22). In this case we know that `option.value` is always unique, so better to use that instead. [Read more](https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318)